careers/careers/templates/careers/position.jinja

```diff
@@ -6,74 +6,46 @@
 {% block body_class %}position-page{% endblock %}
 
 {% block content %}
-<section class="mzp-c-hero mzp-t-dark page-hero">
-  <div class="mzp-l-content">
-    <div class="mzp-c-hero-body">
-      <h1 class="mzp-c-hero-title">Feel good about your work again</h1>
-      <div class="mzp-c-hero-desc">
-        <p>Join us.</p>
+<section class="job-post" id="job-post">
+  <div class="job-row">
+    <div>
+      <div class="job-title">
+        <strong>{{ position.title }}</strong>
       </div>
-    </div>
-  </div>
-</section>
-
-<br />
-
-<h2>{{ position.title }}</h2>
-
-<div class="c-apply-button-container">
-  <a href="{{ position.apply_url }}" class="mzp-c-button">Apply for this job</a>
-  <small>
-    <a rel="external" href="https://assets.mozilla.net/pdf/Mozilla_Applicant_Privacy_Notice_May_2018.pdf">Applicant Privacy Notice</a>
-  </small>
-</div>
-
-<br />
-
-<div class="mzp-l-card-hero">
-  <section class="mzp-c-card mzp-c-card-large job-post" id="job-post" data-location-filter="{{ position.location_filter
-    }}">
-    <div class="mzp-c-card-content">
-      <div class="mzp-c-card-desc">
-        <dl class="mzp-u-list-styled">
+      <div class="title-location">
+        at 
+        {% if position.is_mofo %} 
+          Mozilla Foundation
+        {% else %}
+          Mozilla Corporation
+        {% endif %}
+      </div>
+      <dl class="mzp-u-list-styled">
+        {% if position.is_mofo == False %}
```

Comment:
This is the more Python way to write this. Just a style thing mostly, but also allows it to do the right thing if the value happened to be `None` or `""`.
```suggestion
        {% if not position.is_mofo %}
```